app/policies/action_policy.rb

```diff
@@ -1,6 +1,8 @@
 class ActionPolicy < ApplicationPolicy
   class Scope < ApplicationPolicy::Scope
     def resolve
+      return graphql_id_query if graphql_query_by_id?
+
       if user.params[:request_id]
```

Comment:
@hsong-rh Why not use
```
if graphql_query_by_id?
 graphql_id_query
else if user.params[:request_id]
....
```

Otherwise it looks like an afterthought in the design process

Reply:
@mkanoor I believe `return graphql_id_query if graphql_query_by_id?` is a recommended style rubocop prefers.